.travis.yml

```diff
@@ -1,15 +1,14 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
 language: generic
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
-    - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
```

Comment:
Considering this is only on Travis it probably means you are missing dependencies or some of those dependencies are unsatisfiable. I'll leave this to you to sort out.